packages/utils/feed/markdown.ts

```diff
@@ -0,0 +1,227 @@
+import { Element as DomHandlerElement } from "domhandler";
+import markdownit from "markdown-it";
+import { full as emoji } from "markdown-it-emoji/dist/index.cjs";
+import footnote_plugin from "markdown-it-footnote";
+import { MixedStyleRecord } from "react-native-render-html";
+
+import {
+  neutral17,
+  neutral33,
+  neutral67,
+  neutralA3,
+  neutralFF,
+  primaryColor,
+} from "@/utils/style/colors";
+
+export type ContentMode = "EDITION" | "BOTH" | "PREVIEW";
+
+// The markdownit instance. Used to get the same parameters at Article creation and consultation.
+export const articleMd = markdownit({
+  linkify: true,
+  breaks: true,
+})
+  .use(emoji)
+  .use(footnote_plugin);
+
+// DOM modifications on document, texts, or elements from react-native-render-html.
+// Because react-native-render-html doesn't allow common CSS selectors, we need to style tags using domVisitors callbacks
+export const renderHtmlDomVisitors = {
+  onElement: (element: DomHandlerElement) => {
+    // Removing marginBottom from the child p of blockquote
+    if (
+      element.name === "blockquote" &&
+      element.children &&
+      element.children.length > 0
+    ) {
+      const tagChild = element.children.find((child) => child.type === "tag");
+      // tagChild is a domhandler Node. It doesn't have attribs, but it has attribs in fact (wtf ?)
+      if (tagChild && "attribs" in tagChild) {
+        tagChild.attribs = {
+          style: "margin-bottom: 0",
+        };
+      }
+    }
+  },
+};
+
+// HTML tags styles used by RenderHtml from react-native-render-html
+export const renderHtmlTagStyles: MixedStyleRecord = {
+  body: {
+    color: neutralA3,
+    fontSize: 14,
+    letterSpacing: -(14 * 0.04),
+    lineHeight: 22,
+    fontFamily: "Exo_500Medium",
+    fontWeight: "500",
```

Comment:
should we make a const for those repeated values ?

Reply:
Yes!
https://github.com/TERITORI/teritori-dapp/pull/1451/commits/a58719baa67622b90a9287cd2225b4cdc13176b8